oozie-to-airflow/o2a.py

```diff
@@ -36,6 +39,25 @@ def main():
     if not dag_name:
         dag_name = os.path.basename(input_directory_path)
 
+    conf_path = os.path.join(input_directory_path, CONFIGURATION_PROPERTIES)
+    if not os.path.exists(conf_path):
+        logging.warning(
+            """
+
+#################################### WARNING ###########################################
+
+The '%s' file was not detected in %s.
```

Comment:
I think, you should use a f-string to format string. Please follow a project code style convention.

Reply:
Done